courts_db/utils.py

```diff
@@ -164,15 +165,16 @@ def load_courts_db():
     for k in data:
         # If a child of a parent court - add parent data to child if not present
         # this should allow for streamed down data.  Inheritance
-        if "parent" in k.keys():
-            if not {"dates", "type", "location"} <= set(k.keys()):
-                parent = [x for x in data if x["id"] == k["parent"]][0]
-                if "dates" not in k.keys():
-                    k["dates"] = parent["dates"]
-                if "type" not in k.keys():
-                    k["type"] = parent["type"]
-                if "location" not in k.keys():
-                    k["location"] = parent["location"]
+        if "parent" in k and not {"dates", "type", "location"} <= set(
+            k.keys()
+        ):
+            parent = [x for x in data if x["id"] == k["parent"]][0]
+            if "dates" not in k:
+                k["dates"] = parent["dates"]
+            if "type" not in k:
+                k["type"] = parent["type"]
+            if "location" not in k:
+                k["location"] = parent["location"]
```

Comment:
Fix lots of this one:

```
courts_db/utils.py:168:12: SIM118 Use `key in dict` instead of `key in dict.keys()`
    |
166 |         # If a child of a parent court - add parent data to child if not present
167 |         # this should allow for streamed down data.  Inheritance
168 |         if "parent" in k.keys():
    |            ^^^^^^^^^^^^^^^^^^^^ SIM118
169 |             if not {"dates", "type", "location"} <= set(k.keys()):
170 |                 parent = [x for x in data if x["id"] == k["parent"]][0]
    |
    = help: Remove `.keys()`
```
https://docs.astral.sh/ruff/rules/in-dict-keys/

Plus combining conditions